### ISSUE TITLE
train a model on other split

### ISSUE THREAD

**Issue by xyf159** [Haochen-Wang409/U2PL#50]
I tried to use U2PL to train under PASCAL VOC on different split, I modified n_sup:92 in config.yaml, but the final result is similar to full labeled data, have you encountered such problem?

**Comment by Haochen-Wang409** [comment]
Did you mean that the performance is quite similar to the supervised baseline under a different split?
We did not encounter this issue..

Could you provide your training log or config?

**Comment by xyf159** [comment]
I think I found a problem, I modified n_sup, but only sampled 1464 setting. 
`et random seed to 2
[2022-06-04 13:36:09,727][    INFO] {'criterion': {'kwargs': {'use_weight': False}, 'type': 'CELoss'},
 'dataset': {'batch_size': 2,
             'ignore_label': 255,
             'mean': [123.675, 116.28, 103.53],
             'n_sup': 183,
             'noise_std': 0.1,
             'std': [58.395, 57.12, 57.375],
             'train': {'GaussianBlur': False,
                       'crop': {'size': [321, 321], 'type': 'rand'},
                       'data_list': '../../../../data/splits/pascal/1464/labeled.txt',
                       'data_root': '../../../../data/VOC2012',
                       'flip': True,
                       'rand_resize': [0.5, 2.0]},
             'type': 'pascal_semi',
             'val': {'crop': {'size': [513, 513], 'type': 'center'},
                     'data_list': '../../../../data/splits/pascal/val.txt',
                     'data_root': '../../../../data/VOC2012'},
             'workers': 2},
 'exp_path': '',
 'net': {'decoder': {'kwargs': {'dilations': [12, 24, 36], 'inner_planes': 256},
                     'type': 'u2pl.models.decoder.dec_deeplabv3_plus'},
         'ema_decay': 0.99,
         'encoder': {'kwargs': {'fpn': True,
                                'multi_grid': True,
                                'replace_stride_with_dilation': [False,
                                                                 True,
                                                                 True],
                                'zero_init_residual': True},
                     'type': 'u2pl.models.resnet.resnet101'},
         'num_classes': 21,
         'sync_bn': True},
 'save_path': 'checkpoints',
 'saver': {'pretrain': '', 'snapshot_dir': 'checkpoints'},
 'trainer': {'contrastive': {'current_class_negative_threshold': 1,
                             'current_class_threshold': 0.3,
                             'high_rank': 20,
                             'low_entropy_threshold': 20,
                             'low_rank': 3,
                             'negative_high_entropy': True,
                             'num_negatives': 50,
                             'num_queries': 256,
                             'temperature': 0.5,
                             'unsupervised_entropy_ignore': 80},
             'epochs': 80,
             'eval_on': True,
             'lr_scheduler': {'kwargs': {'power': 0.9}, 'mode': 'poly'},
             'optimizer': {'kwargs': {'lr': 0.0005,
                                      'momentum': 0.9,
                                      'weight_decay': 0.0001},
                           'type': 'SGD'},
             'unsupervised': {'TTA': False,
                              'apply_aug': 'cutmix',
                              'drop_percent': 80}}}
set random seed to 2
[Info] Load ImageNet pretrain from '/home/suibian/xyf/U2PL-main/u2pl/models/resnet101.pth' 
missing_keys:  [] 
unexpected_keys:  ['fc.weight', 'fc.bias']
[Info] Load ImageNet pretrain from '/home/suibian/xyf/U2PL-main/u2pl/models/resnet101.pth' 
missing_keys:  [] 
unexpected_keys:  ['fc.weight', 'fc.bias']
[2022-06-04 13:36:11,307][    INFO] # samples: 1464
[2022-06-04 13:36:11,324][    INFO] # samples: 9118
[2022-06-04 13:36:11,333][    INFO] # samples: 1449
[2022-06-04 13:36:11,334][    INFO] Get loader Done...
[2022-06-04 13:36:11,378][    INFO] # samples: 1464
[2022-06-04 13:36:11,393][    INFO] # samples: 9118
[2022-06-04 13:36:11,405][    INFO] # samples: 1449
[2022-06-04 13:36:11,406][    INFO] Get loader Done...
[Info] Load ImageNet pretrain from '/home/suibian/xyf/U2PL-main/u2pl/models/resnet101.pth' 
missing_keys:  [] 
unexpected_keys:  ['fc.weight', 'fc.bias']
[Info] Load ImageNet pretrain from '/home/suibian/xyf/U2PL-main/u2pl/models/resnet101.pth' 
missing_keys:  [] 
unexpected_keys:  ['fc.weight', 'fc.bias']
[2022-06-04 13:36:12,955][    INFO] The kwargs for lr scheduler: 0.9
[2022-06-04 13:36:12,957][    INFO] The kwargs for lr scheduler: 0.9
`

**Comment by Haochen-Wang409** [comment]
```../../../../data/splits/pascal/1464/labeled.txt``` contains 1464 samples.

**Comment by xyf159** [comment]
Thank you for your reply, my confusion has been solved.
